### PR TITLE
Feature/data card overview page

### PR DESCRIPTION
**Pull request by ARADDCC012**

### CODE REVIEW

File: frontend/pages/entry/new.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        ) : (
+          <Card sx={{ p: 4 }}>
+            <Stack spacing={4} justifyContent='center' alignItems='center'>
+              <Typography variant='h6' component='h1' color='primary'>
+                Create Data Card/Model
+              </Typography>
+              <FileUpload color='primary' fontSize='large' />
+              <Stack
+                direction={{ sm: 'column', md: 'row' }}
+                spacing={4}
```
Comment (ARADDCC002) [inline frontend/pages/entry/new.tsx:33]: Minor and I appreciate this wasn't you originally, but is it worth using `theme.spacing` here? Good opportunity to improve older code

File: frontend/pages/entry/new.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                justifyContent='center'
+                alignItems='center'
+                divider={<Divider orientation='vertical' flexItem />}
+              >
+                <Card
+                  sx={{
+                    width: '300px',
+                    p: 2,
```
Comment (ARADDCC002) [inline frontend/pages/entry/new.tsx:41]: See previous comment about `theme.spacing`

File: frontend/pages/entry/new.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                  }}
+                >
+                  <Stack spacing={2}>
```
Comment (ARADDCC002) [inline frontend/pages/entry/new.tsx:44]: See above

File: frontend/pages/entry/new.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                    >
+                      Create
+                    </Button>
+                  </Stack>
+                </Card>
+                <Card
+                  variant='outlined'
+                  sx={{
+                    width: '300px',
+                    p: 2,
```
Comment (ARADDCC002) [inline frontend/pages/entry/new.tsx:66]: See above

File: frontend/pages/entry/new.tsx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+                  }}
+                >
+                  <Stack spacing={2}>
```
Comment (ARADDCC002) [inline frontend/pages/entry/new.tsx:69]: See above

File: frontend/src/entry/CreateEntry.tsx [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+      const data = await response.json()
+      router.push(`/${toKebabCase(kind)}/${data.model.id}`)
+    }
+  }
+
+  const privateLabel = () => {
+    return (
+      <Stack direction='row' justifyContent='center' alignItems='center' spacing={1}>
+        <Lock />
+        <Stack sx={{ my: 1 }}>
```
Comment (ARADDCC002) [inline frontend/src/entry/CreateEntry.tsx:56]: See above comment about spacing

**Review by ARADDCC002 — CHANGES_REQUESTED** [review]

File: frontend/pages/entry/new.tsx (hunk 1 above)
Comment (ARADDCC012) [reply frontend/pages/entry/new.tsx:33]: This shorthand uses `theme.spacing` under the hood. I've instead removed all unnecessary uses of `theme.spacing()` with the shorthand.

**Review by ARADDCC002 — APPROVED** [review]

**Pull request merged**